### ISSUE TITLE
netloc_draw.html not distributed

### ISSUE THREAD

**Issue by loveshack** [open-mpi/hwloc#285]
The netloc_draw.html file mentioned in the doc for netloc isn't distributed.  I think it just needs

```
--- a/utils/netloc/draw/Makefile.am
+++ b/utils/netloc/draw/Makefile.am
@@ -21,3 +21,5 @@ netloc_draw_to_json_SOURCES = \
 netloc_draw_to_json_LDADD = \
         $(top_builddir)/netloc/libnetloc.la \
                $(top_builddir)/hwloc/libhwloc.la
+
+dist_pkgdata_DATA = netloc_draw.html
```

**Comment by bgoglin** [comment]
Thanks. We also need several related javascript files. @cbordage will look at it to see if we want to distribute this in 2.0 or later (or keep it somewhere online where people could inject their netloc files).

**Comment by cbordage** [comment]
We will fix that in the next tarball. In the meantime, you can use the git version.
Thank you for the report.

**Comment by bgoglin** [comment]
Fixed in master 786bf057b4b975d6092cfe0f5c6dbb2fd27b7f74 and soon in 2.0rc2, thanks!
